app/pages/1_Scenario_comparison.py

```diff
@@ -66,10 +67,9 @@ def main():
             network_map, option
         )
 
-    # TODO fix tech map to match with the Generators or Links
-    #df_techs = [tech_map[c] for c in df.columns]
-    #tech_colors = get_colors_map()
-    #plot_color = [tech_colors[c] for c in df_techs]
+    df_techs = [tech_map[c] for c in df.columns]
+    tech_colors = get_colors_map()
+    plot_color = [tech_colors[c] for c in df_techs]
```

Comment:
Absolutely agree with your suggestion that some grouping could definitely make sense. There may be two tricks to do that:
1) assign the same column names to the technologies to be grouped;
2) use a pre-defined colour order.

Both tricks should be applied to `df` before plotting. You may use Fabian's solution as an inspiration: https://github.com/fneum/spatial-sector-dashboard/blob/be0d521dccdad62e3c17252a10f12cf2dbb3d13a/streamlit_app.py#L107-L113